spec/patterns/httpd_spec.rb

```diff
@@ -2,23 +2,112 @@
 require "spec_helper"
 require "logstash/patterns/core"
 
+describe "HTTPD_COMBINEDLOG" do
+
+  let(:value) { '83.149.9.216 - - [24/Feb/2015:23:13:42 +0000] "GET /presentations/logstash-monitorama-2013/images/kibana-search.png HTTP/1.1" 200 203023 "http://semicomplete.com/presentations/logstash-monitorama-2013/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/32.0.1700.77 Safari/537.36"'}
+
+  it "generates the clientip field" do
+    expect(grok_match(subject, value)).to include(
+      'clientip' => '83.149.9.216',
```

Comment:
++ This works and is super nice! More intel: https://www.relishapp.com/rspec/rspec-expectations/docs/built-in-matchers/include-matcher